.github/workflows/hosting-comment-checker.yml

```diff
@@ -0,0 +1,39 @@
+name: Hosting re-checker
+
+on:
+  issue_comment:
+    types:
+      - created
+      - edited
+
+jobs:
+  hosting:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.comment.body == '/hosting re-check'}}
+    steps:
+      - name: Ack
+        uses: actions/github-script@v4
+        with:
+          script: |
+            const {owner, repo} = context.issue
+            github.reactions.createForIssueComment({
+              owner,
+              repo,
+              comment_id: context.payload.comment.id,
+              content: "+1",
+            });
+      - uses: actions/checkout@v2
+        with:
+          ref: 'master'
+      - uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin' # See 'Supported distributions' for available options
+          java-version: '11'
+          cache: 'maven'
+      - name: Build with Maven
+        run: mvn -B package -Dspotbugs.skip
+      - name: Run hosting checker
+        run: |
+          java -cp target/repository-permissions-updater-1.0-SNAPSHOT-bin/repository-permissions-updater-1.0-SNAPSHOT.jar io.jenkins.infra.repository_permissions_updater.hosting.HostingChecker ${{ github.event.issue.number }}
```

Comment:
not that I think version number will ever change, but you could use * instead of 1.0 right?

Reply:
Probably but it’ll never change